cves/CVE-2016-10192.yml

```diff
@@ -90,14 +98,21 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: a5d25faa3f4b18dac737fdb35d0dd68eb0dc2156
-  note: ''
-- commit: ''
-  note: ''
+  note: "Fix in most recent version"
+- commit: 37904d11779482f375b13da24f33f75daf13638f 
```

Comment:
Can you elaborate on how this commit fixes the vulnerability? I see that there's now an assertion that checks the array length, but some clarification here would be helpful,

Reply:
added clarification for vulnerability fix.